### ISSUE TITLE
[BUG] Import cuDF ValueError

### ISSUE THREAD

**Issue by ritchieng** [rapidsai/cudf#1165]
I'm using CUDA 10.0 Python 3.6 via `pip install cudf-cuda100==0.5 cuml-cuda100==0.5` on Google Colab and I'm facing this error on import via `import cudf`

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-10-e13365c50bc4> in <module>()
----> 1 import cudf

/usr/local/lib/python3.6/dist-packages/cudf/__init__.py in <module>()
      1 # Copyright (c) 2018, NVIDIA CORPORATION.
----> 2 from cudf import dataframe
      3 
      4 from cudf.dataframe import DataFrame, from_pandas
      5 from cudf.dataframe import Index

/usr/local/lib/python3.6/dist-packages/cudf/dataframe/__init__.py in <module>()
----> 1 from cudf.dataframe import (buffer, dataframe, series,
      2     index, numerical, datetime)
      3 
      4 from cudf.dataframe.dataframe import DataFrame, from_pandas
      5 from cudf.dataframe.index import (Index, GenericIndex,

/usr/local/lib/python3.6/dist-packages/cudf/dataframe/dataframe.py in <module>()
     20 
     21 from cudf import formatting, _gdf
---> 22 from cudf.utils import cudautils, queryutils, applyutils, utils
     23 from .index import as_index, Index, RangeIndex
     24 from .series import Series

/usr/local/lib/python3.6/dist-packages/cudf/utils/applyutils.py in <module>()
     11 from cudf.utils.docutils import docfmt_partial
     12 from . import cudautils
---> 13 from cudf.dataframe.series import Series
     14 
     15 

/usr/local/lib/python3.6/dist-packages/cudf/dataframe/series.py in <module>()
     12 from cudf import formatting
     13 from .buffer import Buffer
---> 14 from .index import Index, RangeIndex, as_index
     15 from cudf.settings import NOTSET, settings
     16 from .column import Column

/usr/local/lib/python3.6/dist-packages/cudf/dataframe/index.py in <module>()
     13 from cudf.utils import cudautils, utils
     14 from .buffer import Buffer
---> 15 from .numerical import NumericalColumn
     16 from .column import Column
     17 from .datetime import DatetimeColumn

/usr/local/lib/python3.6/dist-packages/cudf/dataframe/numerical.py in <module>()
     16 from cudf.comm.serialize import register_distributed_serializer
     17 from cudf._gdf import nvtx_range_push, nvtx_range_pop
---> 18 from cudf._sort import get_sorted_inds
     19 
     20 import cudf.bindings.reduce as cpp_reduce

/usr/local/lib/python3.6/dist-packages/cudf/_sort.py in <module>()
      5 from cudf.dataframe.column import Column
      6 from cudf.utils import cudautils
----> 7 import cudf.bindings.sort as cpp_sort
      8 from cudf.dataframe import columnops
      9 from librmm_cffi import librmm as rmm

__init__.pxd in init cudf.bindings.sort()

ValueError: numpy.ufunc size changed, may indicate binary incompatibility. Expected 216 from C header, got 192 from PyObject
```

**Comment by ritchieng** [comment]
Solved it quickly via `pip install numpy==1.16.0`.

**Comment by kkraus14** [comment]
This was fixed in cudf and cuml 0.5.1 as well. Please update your pip install accordingly.
